scripts/dater.rb

```diff
@@ -33,7 +33,7 @@ def date_list
     body = res.body
     json = JSON.parse(body)
     json.shift # pop off header row
-    timestamps = json.map { |archive| Date.parse(archive[1]) }
+    timestamps = json.map { |archive| archive[1] == nil ? Date.now : Date.parse(archive[1]) }
```

Comment:
Use `.nil?` instead? Unless you don't want to because of your ~ ternary operator ~